src/app/components/core/notifications/notification-service-modal/notification-service-modal.component.ts

```diff
@@ -99,6 +105,20 @@ export class NotificationServiceModalComponent implements OnChanges {
     }
   }
 
+  public getAllServices() {
+    this.servicesApiService.getAllServices()
+      .subscribe((res: any) => {
+        this.service = res.services.find(s => s.type === 'Notification');
+      },
+        (error) => {
+          if (error.status === 0) {
+            console.log('service down ', error);
+          } else {
+            this.alertService.error(error.statusText);
```

Comment:
we should not show error, but handle gracefully as schedule could have been disabled & restarted, hence no live/shut-downed service?!